__tests__/app.test.js

```diff
@@ -0,0 +1,40 @@
+const request = require("supertest");
+const app = require("../app.js");
+const connection = require("../db/connection.js");
+const data = require("../db/data/test-data/index");
+const seed = require("../db/seeds/seed.js");
+
+beforeEach(() => {
+  return seed(data);
+});
+
+afterAll(() => {
+  connection.end();
+});
+
+describe("app", () => {
+  describe("/api", () => {
+    test("200: GET responds with a message of connection", () => {
+      return request(app)
+        .get("/api")
+        .expect(200)
+        .then(({ body }) => {
+          expect(body.msg).toBe("Connected");
+        });
+    });
+  });
+  describe("/api/categories", () => {
+    test("200: responds with an array of categories", () => {
+      return request(app)
+        .get("/api/categories")
+        .expect(200)
+        .then((body) => {
+          const categories = body.body;
```

Comment:
The forEach testing is good, however you also should to check the length of the categories. This is because if the categories are the wrong data type, then the tests in the forEach won't fail, and the tests would pass.

Testing for length will make sure that you get an array back, and the the length is correct.